packages/runtime/datastore/src/test/channelStorageService.spec.ts

```diff
@@ -38,16 +39,16 @@ describe("ChannelStorageService", () => {
             id: null,
             trees: {},
         };
-        const storage: Pick<IDocumentStorageService, "read"> = {
-            read: async (id: string) => {
-                return id;
```

Comment:
Hard to say (quickly) how this code was used, but knowing what read() was supposed to return (base64-encoded string), I'd guess that "id" passed here is also base64-encoded payload. If so, should implementation of readBlob assume that is should be decoded from base64? Maybe not - it all depends on expectations of users of this object, but assuming it was passed around (which seems like that case), I'd guess that's the case.
Really ugly if it's really the case :) But it is what it is, we want to keep existing behavior (i.e. safe refactoring preserving existing logic)